docusaurus.config.js

```diff
@@ -76,8 +76,7 @@ const config = {
       title: "hackmud Wiki",
     },
     footer: {
-      copyright: `Copyright © ${new Date().getFullYear()} ComCODE`,
-      links: [],
+      copyright: `Licensed by ComCODE with <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/">CC BY-NC-SA 4.0</a> for creative and educational use.`,
```

Comment:
I'd consider adding `target="_blank"` (so the link opens in a new tab and people don't accidentally leave the wiki) and `rel="noopener noreferrer"` (for security reasons).

Reply:
updated